.github/workflows/add-pr-to-project.yml

```
@@ -2,7 +2,7 @@ name: PR을 프로젝트에 자동 할당합니다.
 
 on:
   pull_request:
-    types: [opened]
+    types: [opened,reopened]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)